react/react1/week1/todo-app/src/components/Todos.js

```diff
@@ -0,0 +1,30 @@
+const todoItem = [
+  {
+    id: 1,
+    activity: 'Get out of bed',
+    dateline: 'Wed Sep 13 2017',
+  },
+  {
+    id: 2,
+    activity: 'Brush teeth',
+    dateline: 'Thu Sep 14 2017',
+  },
+  {
+    id: 3,
+    activity: 'Eat breakfast',
+    dateline: 'Fri Sep 15 2017',
+  },
+];
+
+export function Todos() {
```

Comment:
It is preferable to stick to the same code style. I mean "export". You can see the difference in the picture.
![image](https://user-images.githubusercontent.com/73549460/194079927-a5568e45-9e5f-44e7-8c8c-fbcc74c144b9.png)